corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -20,11 +20,9 @@
 from corehq.util.test_utils import TestFileMixin, softer_assert
 
 
-class SubmissionTest(TestCase):
-    maxDiff = None
-
+class BaseSubmissionTest(TestCase):
```

Comment:
why do these need to be different test classes?

Reply:
Just to keep their intent separate. Initially i put all new 6 tests in `SubmissionTest` but then i thought it would be better to keep them in two different classes based on what they were testing